src/components/Cards/MeetingCard/MeetingCard.stories.tsx

```diff
@@ -21,7 +22,8 @@ meeting.args = {
 
 export const meetingSearchResult = Template.bind({});
 meetingSearchResult.args = {
-  imgSrc: "https://mynorthwest.com/wp-content/uploads/2016/05/arena2-1.jpg",
+  staticImgSrc: "https://mynorthwest.com/wp-content/uploads/2016/05/arena2-1.jpg",
+  hoverImgSrc: "https://mynorthwest.com/wp-content/uploads/2016/05/arena2-1.jpg",
```

Comment:
Since `hoverImgSrc` is apparently supposed to be animated (according to the `MeetingCardProps` comment), was including a `.jpg` here intentional or an accident?

Reply:
Was too lazy to search for an animated img. 😅

Reply:
Lolol I feel that, no worries.